frontend/app/src/components/Sidebar/Sidebar.tsx

```diff
@@ -291,23 +291,20 @@ class Sidebar extends PureComponent<SidebarProps, State> {
     // The tabindex is required to support scrolling by arrow keys.
     return (
       <>
```

Comment:
This update in Sidebar is the real change - rendering logo & expand button based on conditional  `collapsedSidebar` caused a slight flicker when the sidebar was changing states between open & closed (when collapse/expand arrows are clicked). We have now removed the conditional so that these elements remain in place, but the sidebar slides over (covers) them when it is expanded.